grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientBuilder.java

```diff
@@ -393,9 +394,12 @@ public <T> T build(Class<T> clientType) {
         if (!clientInterceptors.isEmpty()) {
             option(INTERCEPTORS.newValue(clientInterceptors));
         }
-        if (exceptionHandler != null) {
-            option(EXCEPTION_HANDLER.newValue(exceptionHandler));
+        if (exceptionHandler == null) {
+            exceptionHandler = DefaultGrpcExceptionHandlerFunction.ofDefault();
```

Comment:
Let's expose `DefaultGrpcExceptionHandlerFunction` using `GrpcExceptionHandlerFunction.ofDefault()`.
```suggestion
            exceptionHandler = GrpcExceptionHandlerFunction.ofDefault();
```

Reply:
Applied feedback. Thanks